assets/blocks/button/color-hooks.js

```diff
@@ -1,6 +1,17 @@
+/**
+ * External dependencies
+ */
```

Comment:
I think this separation is interesting, helping visually when reading components with many imports. It's a standard used in Gutenberg and WooCommerce. Since it doesn't make any harm, I think we could try. And if we identify that it's bothering us, we can remove it easily in the future. =)